File: activityTestCase/app/src/main/java/com/example/activitytestcase/MainActivity.java
package com.example.activitytestcase;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.d("Tag","Ashish is good boy");
        Log.e("Tag","nahi nahi");
        Log.i("Tag","oh no bro");
        Log.v("Tag","now the turn of the verbose megthod");
        try {
            int a=10/0;
            //do something
        } catch (Exception e) {
            Log.d("Tag", "General Exception:\n", e);
        }

    }
}

File: activityTestKotlin/app/src/main/java/com/example/activitytestkotlin/MainActivity.kt
package com.example.activitytestkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        //var l:TextView= R.id.textviewname.text.toString();
        //var edit:TextView = findViewById(R.id.textviewname);
       // var s = edit.text.toString();
    }

    public fun add(name1:String , name2 : String) : String
    {
        var addition = name1 + name2;
        return addition;
    }
}